conformance_tests/sysman/test_sysman_pci/src/test_sysman_pci.cpp

```diff
@@ -171,9 +171,21 @@ TEST_F(PCI_TEST, GivenSysmanHandleWhenRetrievingPCIStatsThenStatsAreReturned) {
     lzt::destroy_command_queue(cq);
     lzt::destroy_command_list(command_list);
 
-    // delay to prevent identical counters in rare cases
-    std::this_thread::sleep_for(std::chrono::seconds(1));
-    auto pci_stats_later = lzt::get_pci_stats(device);
+    zes_pci_stats_t pci_stats_later{};
+    uint32_t wait = 0;
+    do {
+      pci_stats_later = lzt::get_pci_stats(device);
```

Comment:
Can we put this statement at the end in this while loop?
So the sequence would be 
```
get _initial_value
Loop{
wait
get_later_value
}
```
This would allow "`pci_stats_later`" to get new values after getting initial value and not waste one iteration of the loop.

Reply:
Updated!